packages/@o3r/schematics/src/rule-factories/ng-add/index.ts

```diff
@@ -29,13 +32,13 @@ export function ngAddPackages(packages: string[], options?: Omit<NgAddPackageOpt
 
   const getInstalledVersion = (packageName: string) => {
     try {
-      const packageJsonAfterInstall = JSON.parse(readFileSync('./package.json', {encoding: 'utf8'}));
+      const fileSystemPackageJson = JSON.parse(readFileSync(path.join(fsWorkingDirectory, 'package.json'), {encoding: 'utf8'}));
```

Comment:
```suggestion
      const fileSystemPackageJson = JSON.parse(readFileSync(path.posix.join(fsWorkingDirectory, 'package.json'), {encoding: 'utf8'}));
```

Reply:
I am building an actual file system path, so I need to use `path.join`